Shared/UI/Views/DisplayableImageView.swift

```diff
@@ -164,10 +164,9 @@ class DisplayableImageView: View {
 
             guard !Task.isCancelled else { return }
 
-            guard let image = UIImage(data: data) else {
-                self.showResult(for: nil)
-                return
-            }
+            let image = await UIImage(data: data)?.byPreparingForDisplay()
```

Comment:
Setting the displayable image was causing a substantial performance hit. It was especially noticeable on the simulator, but happened on all platforms. iOS 15 introduces this new function that prepares the image for display before we actually tell the UIImageView to display it.